.gitignore

```diff
@@ -1,7 +1,6 @@
 *.profraw
 *.pyc
 source/compile_commands.json
-source/python/.idea
 source/python/neuropod.egg-info/
```

Comment:
why remove this?

Reply:
There is another line below .idea that match all appearances of .idea in repo.